langserver/handler.go

```diff
@@ -589,7 +590,10 @@ func (h *langHandler) handle(ctx context.Context, conn *jsonrpc2.Conn, req *json
 }
 
 func replaceCommandInputFilename(command string, fname string) string {
+	ext := path.Ext(fname)
```

Comment:
Why you don't use filepath.Ext ?

Reply:
I didn't aware of the `filepath.Ext` function. Do you want me to change it over?

Reply:
changed to filepath package instead

Reply:
Hi @mattn , can you check it out again?